prow/bugzilla/types.go

```diff
@@ -150,12 +150,21 @@ type BugUpdate struct {
 // See API documentation at:
 // https://bugzilla.redhat.com/docs/en/html/integrating/api/Bugzilla/Extension/ExternalBugs/WebService.html
 type ExternalBug struct {
```

Comment:
Maybe we should rename this now when it's no longer a generic `ExternalBug` but has GitHub-specific fields?